examples/serve-root.rs

```diff
@@ -25,7 +25,7 @@ fn get_content_type(path: &Path) -> &'static str {
 
 fn main() {
     use ascii::AsciiCast;
-    let server = tiny_http::ServerBuilder::new().with_random_port().build().unwrap();
+    let server = tiny_http::Server::http("0.0.0.0:0").unwrap();
```

Comment:
I know that you didn't change the functionality of this example, but could you change this to not use a random port? It could be the case that the port is less than 1024, which requires super user privileges (at least on unix machines), which could cause the test to fail.